### ISSUE TITLE
[Broken] Unable to extract JS player URL

### ISSUE THREAD

**Issue by GanerCodes** [blackjack4494/yt-dlc#38]
- [X] I'm reporting a broken site support
- [X] I've verified that I'm running youtube-dlc version **2020.10.26**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

```
$ youtube-dlc -g -v https://youtu.be/QpQvJExilsk

[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-g', '-v', 'https://youtu.be/QpQvJExilsk']
[debug] Loading archive file None
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dlc version 2020.10.26
[debug] Python version 3.8.5 (CPython) - Windows-10-10.0.17763-SP0
[debug] exe versions: ffmpeg 4.3.1, ffprobe 4.3.1
[debug] Proxy map: {}
ERROR: Unable to extract JS player URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dlc with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "c:\python38\lib\site-packages\youtube_dlc\YoutubeDL.py", line 830, in extract_info
    ie_result = ie.extract(url)
  File "c:\python38\lib\site-packages\youtube_dlc\extractor\common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "c:\python38\lib\site-packages\youtube_dlc\extractor\youtube.py", line 2065, in _real_extract
    jsplayer_url_json = self._search_regex(
  File "c:\python38\lib\site-packages\youtube_dlc\extractor\common.py", line 1010, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dlc.utils.RegexNotFoundError: Unable to extract JS player URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dlc with the --verbose flag and include its complete output.
```

This started occurring about two days ago and happens the majority of the time when trying to get the URL for a video.

**Comment by blackjack4494** [comment]
Search before posting.

**Comment by nixxo** [comment]
fixed in #35
